### ISSUE TITLE
Cross Entropy Loss Function

### ISSUE THREAD

**Issue by salemmohammed** [apache/incubator-mxnet#5357]
Hi all,

I am trying to run distributed version on image classification python example and I am getting the output. Also, I am trying to measure the loss function for each 20 epochs. However, I could not do it. Can anybody give me a suggestion of who to report loss function after each 20 epochs.

Sincerely

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
